application/templates/entity-gone.html

```diff
@@ -18,18 +18,4 @@ <h1 class="govuk-heading-xl">Entity Removed</h1>
 
   {% block recordEnd %}{% endblock recordEnd %}
 
-  {% block footerStart %}
-    {%- from "digital-land-frontend/components/feedback/macro.jinja" import dlFeedback %}
-    {% set subject = "Feedback on removed entity with number " + entity %}
-    {{ dlFeedback({
-        "text": "Spotted an issue? Let us know so we can improve the data.",
-        "action": {
-            "text": "There is something wrong with the data",
-            "href": "mailto:" + templateVar.email  + "?subject=" + subject
-        },
-        "container": true
-        })
-      }}
-  {% endblock footerStart %}
-
```

Comment:
This was never being shown anyway as the footerStart block isn't declared anywhere down upstream